### PR TITLE
LOTO procedure first draft

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+.. _AuxTel-LOTO-procedure-Overview:
+
+Overview
+^^^^^^^^
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+AuxTel must be under LOTO (lock-out tag-out) any time physical work is undertaken on the mount or the dome. 
+Working with environmental sensors, on the truss, mirrors, or anywhere in the dome where unexpected movement could cause a safety hazard, the telescope must be locked out.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:35]: When working

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+.. _AuxTel-LOTO-procedure-Precondition:
+
+Precondition
+^^^^^^^^^^^^
+
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+Before putting the AuxTel under LOTO, you must have completed the :ref:`LOTO safety training <Daytime-Operations-Safety-Control-Safety-Systems-LOTO>` and familiarized yourself with the :ref:`Control Safety Systems <Daytime-Operations-Safety-Control-Safety-Systems>` document.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:47]: Maybe consider flipping the reference to these links, and linking directly to the LOTO safety training video? Right now, they both pretty much link to the same page. Could also just combine them into one linked statement.

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+Before putting the AuxTel under LOTO, you must have completed the :ref:`LOTO safety training <Daytime-Operations-Safety-Control-Safety-Systems-LOTO>` and familiarized yourself with the :ref:`Control Safety Systems <Daytime-Operations-Safety-Control-Safety-Systems>` document. 
+You should have your own unique lock and key with your information identified, and this work must be coordinated with the day crew. 
+
+- The telescope must be parked in a safe position, and all CSCs in STANDBY. 
+
+    - If the ATDome is needed while the AuxTel is under LOTO, this must be verified with the safety team and day crew.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:52]: Does this mean that the ATDome can be moved while the AuxTel is under LOTO? I think so, so we might need to be more explicit about that early on in the procedure, maybe even in the overview section above.

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+Post-Condition
+^^^^^^^^^^^^^^
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+The drives are power off, and the AuxTel Control cabinet is locked from the outside.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:66]: Typo: "powered", not sure if the comma is needed here either.

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+   b. DRV #2 EL M3 - The drives for elevation and tertiary mirror motion.
+
+   c. DRV #3 NR1 NR2 - Motors for the nasmyth rotators 1 and 2. 
+
+#. Switch off all three brakers by flipping the switch down. 
+
+#. Close the cabinet, and secure your lock and tag in the loop underneath the cabinet door handle. 
+
+#. Verify that the AuxTel will not move intentionally by sending a command with the Engineering User Interface (EUI) on the machine located on the first floor.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:105]: This is kind of picky, but do we have a figure or example to show here? I can imagine that people which are familiar with LOTO might want to use the procedure in the dome, even though they haven't used the EUI before.

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+Troubleshooting
+^^^^^^^^^^^^^^^
+
+.. This section should include troubleshooting information. Information in this section should be strictly related to this procedure.
+
+.. If there is no content for this section, remove the indentation on the following line instead of deleting this sub-section.
+
+No troubleshooting information is applicable to this procedure.
+In the NEARLY IMPOSSIBLE event that the telescope moves after turning off all three brakers, stop work immediately and notifty the day crew, including Jacques Sebag, Mario Rivera, and the electronics team.
```
Comment (edennihy) [inline Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:120]: Typo: "breakers". But I also have to ask, why do we say it is nearly impossible but not impossible? A mechanical event or something?

File: Daytime-Operations/Safety/Control-Safety-Systems.rst [hunk 7]
```diff
@@ -27,6 +27,9 @@ Control Safety Systems
 Presented here is an overview of the concepts LOTO (Lockout/Tagout), GIS (Global Interlock System), and E-STOP (Emergency Stop), and when each procedure or system is used. 
 Specific procedures, like how to LOTO the Telescope Mount Assembly (TMA), or how to reset the GIS to resume operations of a specific component, are not described here. 
 The final procedures for LOTO will be documented on the Observatory Operations pages and linked here. 
+
+    - :ref:`Auxiliary-Telescope-AuxTel-LOTO-procedure`
```
Comment (edennihy) [inline Daytime-Operations/Safety/Control-Safety-Systems.rst:31]: In my build I am not seeing a link here, just the text. I think it's missing the <Daytime-Operations-Auxiliary-Telescope-AuxTel-LOTO-procedure.html>`

**Review by edennihy — APPROVED** [review]
A few minor changes for you to consider before merging. Looks like you will also need to rebase your branch to main.

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 1 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:35]: 👍

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 2 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:47]: 👍

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 3 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:52]: Clarified. 👍

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 4 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:66]: 👍

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 5 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:105]: Figure added and clarified: 👍

File: Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst (hunk 6 above)
Comment (Slimleashma) [reply Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst:120]: Removed "nearly" and just left impossible. But, sometimes the impossible happens, so I left it to say, if you think the LOTO is complete and the telescope still moves, we in trouble. 👍

File: Daytime-Operations/Safety/Control-Safety-Systems.rst (hunk 7 above)
Comment (Slimleashma) [reply Daytime-Operations/Safety/Control-Safety-Systems.rst:31]: 👍

**Comment by edennihy** [comment]
Thanks for all the changes, all comments marked as resolved.

**Pull request merged**